README.md

```diff
@@ -25,7 +36,7 @@ ssmtp is a very lightweight send-only sendmail emulator. Let's see an example of
     # Dockerfile
     FROM helder/php
 
-    RUN apt-install ssmtp && \
+    RUN apt-get install ssmtp -y && \
```

Comment:
There is no error here. This example uses my [helder/php](https://hub.docker.com/r/helder/php/) image, which includes an [apt-install](https://github.com/helderco/docker-php/blob/master/template/bin/apt-install) script created by me.

Reply:
OK, I didn't realize that :)
Nice to learn about it